src/js/filter.ts

```diff
@@ -2,7 +2,22 @@
 
 import type Choices from "choices.js";
 import type { CircleMarker, FeatureGroup } from "leaflet";
-import type { CityId, CityEntry } from "./types";
+import type { CityId, CityEntry, PopulationSliders } from "./types";
+
+const POPULATION_INTERVALS: Array<[string, number]> = [
```

Comment:
No changes made to this.

Reply:
Is there a reason why this is now in filter.ts? Why not import the intervals from populationSlider.ts?

Reply:
It would result in a circular import. `populationSlider.ts` imports from `filter.ts`, so `filter.ts` can't also import from `populationSlider.ts`.